packages/fxa-settings/src/components/LinkedAccounts/index.test.tsx

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
```

Comment:
Hrm I use webstorm, I'll see if there is an equivalent.

Reply:
TBH, it seems like typescript hero is no longer maintained. I've just seen that feedback on other PRs, and it might be int he docs some where. I guess we should just all use the same thing to avoid needless changes on the import statements 🤷 .